tutorxqueue/plugin.py

```diff
@@ -216,10 +235,17 @@ def show_submission(self, queue):
             "return_code": response["return_code"],
         }
 
-    def count_submissions(self, queue):
+    def count_submissions(self, queue: str) -> Any:
```

Comment:
Thanks for taking the time to type all those functions, I understand it's not an easy job, nor a very fun one.

Reply:
Anytime!